scheduler/daemon/host/host_manager.go

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Unified naming scheduler/daemon/host/host_manager.go and scheduler/daemon/host_mgr.go